resources/measures/nighttime_oa_controls/measure.rb

```diff
@@ -0,0 +1,200 @@
+      return true
+    end
+
+    ##AA take out these counts if not needed 
+    oa_schd_1_count = 0
+    oa_schd_op_count = 0
+    op_schd_1_count = 0
+    fan_schd_1_count = 0
+    fan_sch_op_count = 0
+
+    # make changes to unitary systems
+	#need to deal with non unitary systems, too 
+    li_unitary_systems.sort.each do |air_loop_hvac|
+
+      # change night OA schedule to match hvac operation schedule for no night OA
+        # Schedule to control whether or not unit ventilates at night - clone hvac availability schedule
+        next unless air_loop_hvac.availabilitySchedule.clone.to_ScheduleRuleset.is_initialized
+        air_loop_vent_sch = air_loop_hvac.availabilitySchedule.clone.to_ScheduleRuleset.get
+        air_loop_vent_sch.setName("#{air_loop_hvac.name}_night_novent_schedule")
+        next unless air_loop_hvac.airLoopHVACOutdoorAirSystem.is_initialized
+        air_loop_oa_system = air_loop_hvac.airLoopHVACOutdoorAirSystem.get.getControllerOutdoorAir
+        next unless air_loop_oa_system.minimumOutdoorAirSchedule.is_initialized
+        air_loop_oa_system.setMinimumOutdoorAirSchedule(air_loop_vent_sch)
+        oa_schd_op_count += 1
+      
+	  
+	   # change fan operation schedule to new clone of hvac operation schedule to ensure night cycling of fans (removing any constant schedules)
+        # Schedule to control whether or not unit ventilates at night - clone hvac availability schedule
+        next unless air_loop_hvac.availabilitySchedule.to_ScheduleRuleset.is_initialized
+        air_loop_fan_sch = air_loop_hvac.availabilitySchedule.clone.to_ScheduleRuleset.get
+        air_loop_fan_sch.setName("#{air_loop_hvac.name}_night_fancycle_schedule")
+        # Schedule to control the airloop fan operation schedule
+        air_loop_hvac.supplyComponents.each do |component|
+          obj_type = component.iddObjectType.valueName.to_s
+          case obj_type
+          when 'OS_AirLoopHVAC_UnitarySystem'
+            component = component.to_AirLoopHVACUnitarySystem.get
+            component.setSupplyAirFanOperatingModeSchedule(air_loop_fan_sch)
+            fan_sch_op_count += 1
+          when 'OS_AirLoopHVAC_UnitaryHeatPump_AirToAir'
+            component = component.to_AirLoopHVACUnitaryHeatPump_AirToAir.get
+            component.setSupplyAirFanOperatingModeSchedule(air_loop_fan_sch)
+            fan_sch_op_count += 1
+          when 'OS_AirLoopHVAC_UnitaryHeatPump_AirToAir_MultiSpeed'
+            component = component.to_AirLoopHVACUnitaryHeatPumpAirToAirMultiSpeed.get
+            component.setSupplyAirFanOperatingModeSchedule(air_loop_fan_sch)
+            fan_sch_op_count += 1
+          when 'OS_AirLoopHVAC_UnitaryHeatCool_VAVChangeoverBypass'
+            component = component.to_AirLoopHVACUnitaryHeatCoolVAVChangeoverBypass.get
+            component.setSupplyAirFanOperatingModeSchedule(air_loop_fan_sch)
+            fan_sch_op_count += 1
+          end
+        end
+    end 
+	
+	#handle non-unitary systems 
```

Comment:
Do we also need to change the fan operation so that we have cycling rather than constant fan operation?

Reply:
Good point--I added that.